lib/scoped_model.dart

```diff
@@ -54,9 +54,12 @@ abstract class Model extends Listenable {
 
   /// Should be called only by [Model] when the model has changed.
   @protected
-  void notifyListeners() {
+  void notifyListeners({bool forced = false}) {
     // We schedule a microtask to debounce multiple changes that can occur
     // all at once.
+    if (forced) {
```

Comment:
It looks like this will cause listeners to be called twice: First immediately and then after a microtask is executed. If `forced = true` is set, the listeners should be called immediately and then the function should exit.